controls/chat/styling/buttons.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Buttons
+page_title: .NET MAUI Chat Documentation - Buttons Styling
+description: Learn how to style the buttons inside the Telerik UI for .NET MAUI Chat control.
+position: 6
+tags: .net maui, telerik .net maui, ui for .net maui, chat, style
+slug: chat-buttons-styling
+---
+
+# .NET MAUI Chat Buttons Styling
+
+This section describes how to style the send, more and speech to text buttons inside the `RadChat` control.
+
+## Style the Send Button
+
+To customize the icon of the send button, use the `SendMessageButtonImage` (`ImageSource`) property.
+
+The example below shows how to customize the Send button in your Chat control.
+
+```XAML
+<FontImageSource x:Key="SendIconSource"
+                 FontFamily="Your font family name"
+                 Glyph="icon code"
+                 Color="Black"
+                 Size="16" />
+```
+
+```XAML
+ <telerik:RadChat x:Name="chat"
+                  SendMessageButtonImage="{StaticResource SendIconSource}"/>
+```
+
+## Style the More Button
+
+The More button provides additional options for the user, such as attaching files or accessing other functionalities.
+
+You can configure and style the More button using the following properties:
+
+* `IsMoreButtonVisible` (`bool`) property controls the visibility of the more button. The default value is `false`.
```

Comment:
Align the style and phrasing of the bullets. Right now they are harder to scan through and to grasp what's going on.